card/mod/bootstrap/db/migrate_core_cards/lib/skin.rb

```diff
@@ -15,9 +15,16 @@ def themes
       json = File.read File.join(vendor_path, "bootswatch", "docs", "api", "4.json")
       JSON.parse(json)["themes"].map { |theme| theme["name"] }
     end
```

Comment:
I should make sure I understand how this lib works.  Is it ever auto loaded?

Reply:
I think the root lib directory is in the autoload paths but this one shouldn't be there